controller_coverage.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Usage: bash ./controller_coverage.sh
+# Then run: firefox controller_coverage/index.html
```

Comment:
Nit: This script could try to open this file (I think using https://portland.freedesktop.org/doc/xdg-open.html on Linux and `open` on macos) if you like?  Or it could at least echo the path of the file to open?